netbout-web/src/main/js/bout.js

```diff
@@ -69,31 +69,51 @@ $(document).ready(
               dataType: 'xml',
               method: 'GET',
               success: function (data) {
-                var appendix = '';
+                var appendix = '', $data = $(data), messagesAsXML = $data.find('message'),
+                    messages = $data.find('#messages');
                 more = '';
-                $(data).find('message').each(
-                  function (idx, msg) {
-                    var $msg = $(msg);
-                    appendix += [
-                      '<div class="message" id="msg',
-                      $msg.find('number').text(),
-                      '"><div class="left"><img class="photo" src="',
-                      $msg.find('link[rel="photo"]').attr('href'),
-                      '"/>',
-                      '</div><div class="right"><div class="meta"><strong>',
-                      escapeHTML($msg.find('author').text()),
-                      '</strong> said ',
-                      escapeHTML($msg.find('timeago').text()),
-                      '</div><div class="text">',
-                      $msg.find('html').text(),
-                      '</div></div></div>'
-                    ].join('');
-                    more = $msg.find('link[rel="more"]').attr('href');
-                  }
-                );
-                $tail.removeAttr('id');
-                $tail.html(appendix + '<div id="tail"/>');
-                $box.attr('data-more', more);
+                if (messagesAsXML) {
+                  messagesAsXML.each(
+                    function (idx, msg) {
+                      var $msg = $(msg);
+                      appendix += [
+                        '<div class="message" id="msg',
+                        $msg.find('number').text(),
+                        '"><div class="left"><img class="photo" src="',
+                        $msg.find('link[rel="photo"]').attr('href'),
+                        '"/>',
+                        '</div><div class="right"><div class="meta"><strong>',
+                        escapeHTML($msg.find('author').text()),
+                        '</strong> said ',
+                        escapeHTML($msg.find('timeago').text()),
+                        '</div><div class="text">',
+                        $msg.find('html').text(),
+                        '</div></div></div>'
+                      ].join('');
+                      more = $msg.find('link[rel="more"]').attr('href');
+                    }
+                  );
+                  $tail.removeAttr('id');
+                  $tail.html(appendix + '<div id="tail"/>');
+                  $box.attr('data-more', more);
+                }
+                if (messages) {
+                  messages.find('.message').each(
+                    function (idx, line) {
+                      var temp = $('<div></div>'), $msg = $(line),
```

Comment:
@erimerturk Let's avoid declaring multiple variables on the same line as it makes the code hard to read.